lib/open_mic.rb

```diff
@@ -0,0 +1,27 @@
+class OpenMic
+  attr_reader :location, :date, :performers
+
+  def initialize(event)
+    @location = event[:location]
+    @date = event[:date]
+    @performers = []
+  end
+
+  def welcome(user)
+    @performers << user
+  end
+
+  def repeated_jokes?
+    jokes = []
+    @performers.each do |performer|
+      performer.jokes.each do |joke|
+        if !jokes.include?(joke)
+          jokes << joke
+        elsif jokes.include?(joke)
+          return true
+        end
+      end
+    end
+    false
```

Comment:
good work here, we'll talk about a way to shorten this in the walkthrough; I really like how you shortcut processing any further with an explicit `return` on line 21